moped-editor/src/layouts/DashboardLayout/NavBar/MobileDropdownMenu.js

```diff
@@ -99,6 +99,15 @@ const MobileDropdownMenu = () => {
             ))}
           </div>
         )}
+        <MenuItem
+          onClick={() => {
+            handleMobileClose();
+            navigate("/moped/account");
+          }}
+        >
+          Account
+        </MenuItem>
+        <MenuItem onClick={() => navigate("/moped/logout")}>Logout</MenuItem>
```

Comment:
I'm not sure about the placement of this being below the "add new project" button in the dropdown, but "Logout" being last feels right to me

Reply:
That feels right to me too. The hamburger is such a nice improvement! ✨